src/core/Directus/Services/TablesService.php

```diff
@@ -1321,9 +1321,7 @@ protected function updateTableSchema(Collection $collection, array $data)
         $this->updateColumnsRelation($name, array_merge($toAdd, $toChange));
 
         foreach ($fields as $fieldData) {
-            if ($fieldData['type'] != DataTypes::TYPE_O2M) {
```

Comment:
Isn't the O2M check important here? I remember the issue we had where we tried setting `not null` on a non-existing column.

Reply:
Yes, we don't need this condition now as `O2M` is an alias type and based on [this](https://github.com/directus/api/blob/62c65dd96d0ba42c5e7f5e6c4a9d8212fbe7c098/src/core/Directus/Database/Schema/SchemaFactory.php#L103) code it'll return.

Reply:
Ok! :sligh